src/modules/translation/languages/en.json

```diff
@@ -41,22 +41,20 @@
   },
   "shortcuts_modal": {
     "title": "Shortcuts",
-    "basic_controls": {
-      "shortcut_reference": "Shortcut reference",
-      "title": "Basic Controls",
-      "undo": "Undo",
-      "redo": "Redo",
-      "reset": "Reset item",
-      "duplicate": "Duplicate item",
-      "delete": "Delete item",
-      "reset_camera": "Reset camera",
-      "zoom_in": "Zoom in",
-      "zoom_out": "Zoom out"
-    },
-    "edit_controls": {
-      "title": "Edit controls",
-      "move": "Move item",
-      "rotate": "Rotate item"
-    }
+    "basic_controls": "Basic Controls",
```

Comment:
why are these "normalized"?

Reply:
i flattened it because i had to move some items from one group to the other and i figured that it doesn't have to be nested, they are all part of shortcuts modal

Reply:
Do you mind the EditProjectModal validation locale being nested? We may have an inconsistency